crates/apub/src/api/mod.rs

```diff
@@ -14,13 +18,17 @@ fn listing_type_with_default(
   type_: Option<ListingType>,
   local_site: &LocalSite,
   community_id: Option<CommunityId>,
-) -> Result<ListingType, LemmyError> {
+) -> ListingType {
```

Comment:
Seems like rust should've warned us about this one. Using a Result here was pointless since there's no possibility for errors.